HISTORY.md

```diff
@@ -1,5 +1,7 @@
 # Rocksdb Change Log
 ## Unreleased
+### Bug Fixes
+* In WritableFileStringStreamAdapter::overflow(), not only we need to handle `std::endl`, but also there are other cases like unsigned long long that calls `os.put()` directly then drops into sputc() function. The internal implementation of `sputc()` calls `overflow()`. As buffering is explicitly disabled in this stream wrapper, every write, not captured by xsputn, becomes an overflow here. Now it is fixed so all characters that are not captured by xsputn will be appended here, unless the file appending fails.
```

Comment:
Can we write this more in terms of user-visible APIs and what is the user impact? It appears `WritableFileStringStreamAdapter` is used only in `BlockBasedTable::DumpTable()`. Then, I believe "dumping block-based SST files in a human-readable format" would be affected. Regarding user impact, it seems the table dump would be truncated?

Reply:
Yes correct, from user side I observed the table dump was truncated in some cases. I have updated the unreleased note accordingly. Thanks for suggestion!